### ISSUE TITLE
Citizens for ACH?

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3366]
http://www.citizensbank.com/commercial/automated-clearing-house-ach.aspx
http://www.citizensbank.com/commercial/accessglobal-ach.aspx
https://www.citizensbank.com/commercial-banking/banking-needs/electronic-payments.aspx

**Comment by whit537** [comment]
I called the 877 number and apparently we don't have the right type of account(?). They're having someone else call me back. I think I'm going to stop in the local office to talk with my contact there.

**Comment by whit537** [comment]
Called my contact at Citizens' local office and she's looking into this for me.

**Comment by whit537** [comment]
"There's an underwriting process, it's not an instant sign-up."

**Comment by whit537** [comment]
Local contact has me in the system and I'm waiting to hear back from the next level up.

**Comment by whit537** [comment]
Just had a call with the next level up. She's going to get me a pricing sheet in the next day or so but sounds like they're offering 20¢ per. I did specify that we want to upload NACHA files programmatically and I think she took that on board (good look on recommending we make that explicit, @sbrendtro). I also emphasized that we're only doing credits, and aren't asking to be underwritten for debits at this point (maybe in the future). Seems to be moving along!

**Comment by sbrendtro** [comment]
As you get talking to more technical people at Citizens, you'll want to get a hold of their specifications for file transmission, file confirmations, and their NACHA format guide.  The NACHA format itself is standard, but each bank seems to like to customize the values transmitted in some of the fields.  And the file confirmation format is completely different for each bank.  All of this information will be needed to build an OpenACH bank plugin specific to Citizens.

**Comment by whit537** [comment]
Meeting next Thursday at 10am to discuss.

**Comment by whit537** [comment]
Per https://github.com/gratipay/gratipay.com/issues/3390#issuecomment-100531709, ask Citizens about bank transfers outside the U.S.

**Comment by sbrendtro** [comment]
FYI, international ACH transfers are available through side banks, but to
my knowledge the country lost no longer includes Euro countries since 2009
and the introduction of SEPA payments.  The transfers could likely be done
via:

1. Bank wire (FedWire) - expensive
2. SEPA - needs euro bank relationship
3. OCT transactions to debit cards (direct to card loads) - Citibank does
these and probably others, but not sure who. Read more here:
https://developer.visa.com/vpp/documents/json/Original_Credit_Transaction__OCT_.html

**Comment by whit537** [comment]
Thanks @sbrendtro. I believe Citizens is owned by RBS, so they may be able to help us with a European bank account that we could use for SEPA.

**Comment by whit537** [comment]
Reticketed from https://github.com/gratipay/gratipay.com/pull/3408/files#r30277011: ask about terms of service to incorporate in ours.

**Comment by whit537** [comment]
Meeting went well. Lots of info but I'm running hard on #3415. We should see a proposal tomorrow, will need to make a decision tomorrow to make this happen in time (guy is on vacation next week, needs to get the ball rolling before he leaves).

**Comment by whit537** [comment]
I pulled the trigger on this. It'll be an adjustment because the fee structure and API are more complicated. One interesting development: we'll have access to 25 countries via [IAT](https://www.frbservices.org/eventseducation/education/fedach_iat_resource_center.html) (cf. https://github.com/balanced/balanced-api/issues/44). We've actually had no fee on payouts since Balanced made an exception for us as an early adopter. Given the more complicated fee structure here, it'll take some work to figure out how we're going to pass that through; reticketed as #3443.

**Comment by whit537** [comment]
- Their [proposal](https://www.dropbox.com/s/3gz79wvfnmiljfr/Gratipay-%20DT.pdf?dl=0) for the "file gateway" (sftp/scp).
 - An [alternate proposal](https://www.dropbox.com/s/oz11ofcb4kad80p/Gratipay%20-%20AMM.pdf?dl=0) for their web-based system.
 - their [IAT guide](https://www.dropbox.com/s/n75p113l3r502gs/AccessGLOBAL_ACH_Guide.pdf?dl=0) (28 pp.):
 - the list of [countries](https://www.dropbox.com/s/mbjiblt74txd67v/FedGlobal%20ACH%2025%20countries%20sheet.ppt?dl=0) for IAT (in PPT, natch)
 - their [NACHA format guide](https://www.dropbox.com/s/21w1gtcetvo9et0/NACHA_FORMAT_LNA_and_SFB.pdf?dl=0)

I told him we would move forward with the file gateway proposal, except for Achieve Access ($50/mo). Not having Achieve Access means:

 - We have to phone in control totals vs. using a web+securid-based system.
 - We have to monitor our account every day for ACH debit fraud. We have 24 hours to report fraud. With Achieve Access we would get to approve all ACH debits, whitelist, and set caps.
 - Direct transmissions should be under dual control, of course.
 - We should look into ACH/cyberattack insurance.

Should I ask him to add Achieve Access after all?

**Comment by whit537** [comment]
I've said yes to Achieve Access at $25/mo. I've named @clone1018 as the secondary employee for dual-control purposes.

**Comment by whit537** [comment]
##### Countries

Austria • Belgium • Canada • Cyprus • Czech Republic • Denmark • Finland• France • Germany • Greece • Ireland • Italy • Luxembourg• Malta • Mexico • Netherlands • Panama • Poland • Portugal • Slovakia • Slovenia • Spain • Sweden • Switzerland • United Kingdom

**Comment by whit537** [comment]
We're expecting to have a new account opened and to hear back next week with next steps.

**Comment by whit537** [comment]
We have a new analyzed checking account open at Citizens. I need to put a little cash in there today. This will replace the New Alliance account.

We are getting a second account at Citizens, a money market account that will have 1% guaranteed interest for 12 months. This will replace our old escrow cold storage account at Ally. It has a $25,000 minimum. Depending on how #1383 goes, we may need to close this out if we're able to drop our escrow low enough.

**Comment by whit537** [comment]
Just double-checked: there's no penalty if we close the money-market account before the 12 months is up or anything like that.

**Comment by whit537** [comment]
I've started connecting New Alliance at Citizens so we can transfer funds online for the money market account. Awaiting verification deposits.

I'll probably just withdraw cash and walk it down the street for the escrow checking account, because that can be a much smaller amount.

**Comment by whit537** [comment]
Transferred $100 into the escrow checking account yesterday. Awaiting verification deposits to enable online transfer before opening up the money market account.

**Comment by whit537** [comment]
Had a long call with our implementation manager to go over the documents she sent me. We have a weekly call scheduled for Thursdays to stay on track. Deliverables from us are executed documents and a NACHA file in [this format](https://www.dropbox.com/s/21w1gtcetvo9et0/NACHA_FORMAT_LNA_and_SFB.pdf?dl=0). We need to make sure we're using the right [SEC codes](http://en.wikipedia.org/wiki/Automated_Clearing_House#SEC_codes). Target at this point is to process on their system for the first time on June 11(!), with June 18 as our fall-back.

**Comment by whit537** [comment]
Average implementation time is 26 days, June 11 is 21 days, June 18 is 28.

**Comment by whit537** [comment]
Do you have a fax number, @clone1018? They need one on the application even though we don't have to actually use it.

**Comment by clone1018** [comment]
@whit537 I have a personal work fax I'll email you.

**Comment by whit537** [comment]
Awesome, thanks! :D

**Comment by whit537** [comment]
> We need to make sure we're using the right [SEC codes](http://en.wikipedia.org/wiki/Automated_Clearing_House#SEC_codes).

Our treasury services rep at Citizens recommends "IIT [Includes CCD, CTX, IAT & PPD]." I would expect us to start out using CCD for paying out team owners, and PPD for paying out team members. As we strengthen our identity verification (#3289), we should see how that affects this (what about a team payout to a member who is actually a corporation and not an individual?). For cross-border payments we'll use IAT, of course.

**Comment by whit537** [comment]
To: Citizens

> I've read and executed the following:
>
 - Certificate of Authority
 - Cash Management Service Order
 - ACHieve Service Request
 - ACHieve Single Administrator Request
 - Challenge Information Form (Chad)
>
Additionally, I have a public key ready for the Data Transmission Questionnaire.
>
A question on the Certificate of Authority: Gratipay is a single-member LLC that is treated as a sole proprietorship by the IRS. Therefore, should I add my SSN to the Certificate?
>
For the ACH Service Request, we *have* identified a fax number, but I am awaiting confirmation of Luke's current mailing address. I am also waiting for Luke's Challenge Information Form.
>
I understand we're supposed to send one email with all of the documents, so I will wait until I have everything before I send it all over. Please expect those from me on Wednesday morning at the latest.

**Comment by whit537** [comment]
> Awaiting verification deposits to enable online transfer before opening up the money market account.

New Alliance account verified at Citizens.

**Comment by whit537** [comment]
To: Citizens

> Please find attached our executed documentation. Let me know if anything is missing.
>
I will follow up with the Challenge Forms, Data Transmission Questionnaire, and public key in a separate email.

To: Citizens

>Here are the Challenge Forms for Luke and myself, as well as the Data Transmission Questionnaire and our public key.
>
I believe that's everything you need from us at this point, yes?

**Comment by whit537** [comment]
From: Citizens

> I have the required agreements from you.   But I was not able to open the attachment with the Public Key.  Are you able to copy the key into an email and send that to me?
>
The last piece of the puzzle is an internal document that I need from your banker.  I will begin the implementation as soon as I receive the internal document.
>
Let me know if you have any questions.

To: Citizens

> Excellent. Here is the public key: [snip]

**Comment by whit537** [comment]
Let's consider this ticket closed when the "internal document" clears. Then we can move to #3488.

**Comment by whit537** [comment]
Had our weekly call with Citizens. The "internal document" is clearance from their underwriting department, who have bumped us to their AML department. In other words, https://github.com/gratipay/gratipay.com/issues/417#issuecomment-78266767 all over again. :-(

You know what? https://github.com/gratipay/inside.gratipay.com/issues/119

**Comment by whit537** [comment]
To: Citizens

> Thanks for the call this morning. I understand that Gratipay needs to clear an underwriting hurdle with regard to third-party processors as well as AML before we have the green light to proceed. Per our call:
>
We show up on bank statements as GRATIPAY.COM plus our support phone number.
>
Here is a list of [our current clients](https://gratipay.com/explore/teams/). Ball is in my court to get you a list of past clients for context (as you know we're in the midst of a major upgrade unrelated to our processing infrastructure migration and we're having to re-sign up our users).
>
On the AML documentation, I've asked [] in a follow-up phone call for a day to review and possibly refine our current practices in an effort to put our best foot forward with Citizens' AML department.
>
Please expect to hear from me again tomorrow.

**Comment by whit537** [comment]
Online transfer won't clear in time. If we're going to get this 1% rate locked in for the money market account, I need to go to New Alliance now and get a cashier's check and walk it down to Citizens and ask for next-day availability.

**Comment by whit537** [comment]
Okay! Money market account opened. O.o

**Comment by whit537** [comment]
:bank: :bike: :moneybag: :bank:

**Comment by whit537** [comment]
@clone1018 has agreed to take a first pass at https://github.com/gratipay/inside.gratipay.com/issues/119 tonight, and I've approached @copiesofcopies about writing an opinion letter to Citizens to accompany the AML documentation.

**Comment by whit537** [comment]
Going to hold off on the opinion letter and see if we can schedule a call with the decision makers at Citizens if needed after they've had a chance to review what @clone1018 comes up with for an AML program. This is in line with Citizens' suggestion to me earlier today.

**Comment by whit537** [comment]
I need to get back to Citizens today about our AML program. Since we're down to the wire for implementation I don't think we can block on https://github.com/gratipay/inside.gratipay.com/issues/119. I think we should give them as much information as exists today, and ask if we can proceed with implementation in parallel with finalizing compliance.

**Comment by clone1018** [comment]
Ouch, I had no idea you needed it by today, that would have been
impossible. Sorry for the misunderstanding.

**Comment by whit537** [comment]
@clone1018 No, it's fine. I just had a call with our sales rep. I'm going to write something up describing where we stand today and the improvements we're planning for the future. He's going to hand that up the ladder, and we'll touch base next week. Outcomes I see are:

 1. They accept what we're able to come up with today and green-light us.
 2. They want to talk further, but are willing to let implementation proceed in parallel with underwriting.
 3. They want to talk further, and block implementation on underwriting.
 4. They reject us.

(1) is great, (2) would be a relief at this point. (3) would put us in a[nother] crunch, and (4), well ...

**Comment by clone1018** [comment]
Couple of important notes from: http://www.fatf-gafi.org/media/fatf/documents/recommendations/pdfs/FATF_Recommendations.pdf

P#14D10: No anonymous accounts or fake names? May not apply.

P#63H15: Enhanced CDD measures for "high risk", mentions geographic risk factors?

P#65H17c: countries with effective AML systems are low risk

P#65H20: Enhanced CDD measures

P#65H21: Simplified CDD measures (low risk) **what we should be doing**

**Comment by whit537** [comment]
@clone1018 Let's use https://github.com/gratipay/inside.gratipay.com/issues/119 for that.

**Comment by whit537** [comment]
To: Citizens

> As you know, Gratipay's risk profile is actively changing right now, and we're doing our best to assess and respond to these changes quickly and carefully. Here is our risk management program as it stands today, which includes our AML program:
>
http://inside.gratipay.com/howto/manage-risk
>
We would welcome a conversation with Citizens' underwriters to further discuss any questions or issues related to our risk program.
>
Thanks, []!

**Comment by whit537** [comment]
From: Citizens

>Thanks Chad
>We’ll be sure to update you with any feedback as soon as we get it. I anticipate [] and I will re-group with the AML and underwriting group now that we have those items they were most interested in to complete their due diligence.

To: Citizens

>Thanks, []. After spending time with this over the weekend, I'm confident we can find a way forward together. I look forward to speaking with you on our Thursday call if not before, and I look forward to your decision to proceed!

**Comment by whit537** [comment]
@clone1018 Will you be able to join our weekly call at 11:30 AM US/Eastern today?

**Comment by clone1018** [comment]
Sure, @whit537 it's in an hour right?

**Comment by whit537** [comment]
@clone1018 Yes, but actually, game off. I got a call this morning that we've been rejected.

**Comment by whit537** [comment]
The two reasons Citizens cited for rejecting us were:

 - We might be a "money exchanger," because we support bitcoin. We [don't support bitcoin]() under the new TOS, and we took it off the FAQ in #3429. I'm guessing they saw bitcoin addresses listed on profiles.
 - We don't do [sanction screening](https://sdnsearch.ofac.treas.gov/), which is true. Reticketed as https://github.com/gratipay/inside.gratipay.com/issues/239.

I was speaking with a salesperson who was proxying about eight risk officers from an internal email thread. "You've even attracted the attention of the regulations and compliance department, which sits above AML." I've asked if we could have a conversation with one or more risk officers to understand their concerns, since that will be helpful in approaching PNC (#3518). Citizens [went public](https://www.citizensbank.com/about-us/news/citizens/2014/9-23-14-IPO-stock-pricing.aspx) last year, and that doesn't make them less skittish. The salesperson referred to "finable offenses," and in particular transactions involving sanctioned parties. It was unclear to me whether the identity of the sender or the receiver was the thing that mattered (cf. Transpay at https://github.com/gratipay/gratipay.com/issues/417#issuecomment-54667221).

Also, they noticed that Kiberpipa is in Slovenia. "Are they a registered charity? How do they know?" cc: @silverhook @domenkozar :)

**Comment by domenkozar** [comment]
We're an NGO. No such thing as charity here, you can just upgrade NGO status into "public benefit" (means just less taxes), which we're in process now-ish.

**Comment by silverhook** [comment]
TL;DR: we’re a registered NGO NPO and currently in the process of gaining the status of an association in public interest (equivalent to the common law charitable organisation).

We fall under the [Societies Act](http://www.pisrs.si/Pis.web/pregledPredpisa?id=ZAKO4242) as „društvo“ (“association”), which is as @domenkozar mentioned by definition (in said act) a non-government, non-profit organisation, based on volunteer work and with a voluntary membership.

LUGOS’ [statute](http://www.lugos.si/drustvo/drustvo/statut) in §8 sets forth as our main activities: 

> - organising social activities for computer users and **mutual help**;
- **education** (organising courses, conferences and workshops);
- **spreading GNU/Linux** into academic and educational organisations, as well as educating potential users everywhere where computers are being used;
- **publishing** all sorts of publications on how to use Unix, Linux and applications for them (lately this is mostly [videos][] online, but the first Slovenian book on GNU/Linux was also authored by LUGOS);
- **translating and localising** certain parts of GNU/Linux, where justifiable and where the need and demand from users arises;
- **fight against the monopoly** on the Slovenian IT market regarding operating systems.

We are currently in the process of gaining the status of „društvo v javnem interesu“ (“society in public interest”), which is the local (perhaps even civil law) equivalent of a charitable association. For this there are some stricter rules you have to meet. We do, but so far never bothered to apply. Once we get the status, the entity and everything stays the same, just in the public registry we hold that additional status.

[videos]: http://video.kiberpipa.org

**Comment by whit537** [comment]
Thanks for the info, @silverhook @domenkozar. I'm afraid it's too late for this round, but it looks like collecting that kind of info could be important for the next round.

**Comment by whit537** [comment]
To: Citizens

> Thanks for your hard work on our behalf, and I'm sorry it didn't come together. I wanted to follow up with an email to address the feedback I heard from you, [], about Gratipay's risk program:
>
 - **We may be regulated by FinCEN.** We rewrote our [terms of service](https://gratipay.com/about/policies/terms-of-service) a month ago specifically to fit the payment processor exemption to the definition of a money transmitter ([31 C.F.R. § 1010.100(ff)(5)(ii)(B)](http://www.ecfr.gov/cgi-bin/text-idx?node=se31.3.1010_1100)). We stopped processing payments under our old terms of service and are rebuilding our customer base from scratch under our new terms. We are well aware of the issues here, and I can confidently say that we are not a money services business and are not regulated by FinCEN.
 - **We process bitcoin.** In fact, we do not process bitcoin. We allow our users to list a bitcoin address on their public profile, but we do not use the address to directly facilitate any transactions on their part. We specifically *removed* this functionality when we rewrote our terms and started rebuilding our user base.
 - **We do not perform sanction screening.** This is true. However, we are happy to add this to our risk program, both for existing and future customers.
>
I understand that this is too little, too late, but I wanted to at least clear up for the record the misconceptions about our MSB status as well as bitcoin. Thanks again for all your hard work, and best wishes.

**Comment by whit537** [comment]
From: Citizens

> Thank you for your email – I will keep this on file in the event we try to form a business relationship in the future.
>
It’s been a pleasure meeting you – I hope you find a good fit for ACH services for your company soon so that your business isn’t interrupted. In the event I hear about your accounts needing to close I will let you know but you may get a letter in the meantime.

**Comment by whit537** [comment]
I got a call from our Citizens account rep, and this letter in the mail:

![Letter from Citizens](https://cloud.githubusercontent.com/assets/134455/8267335/7e5f4a0a-172a-11e5-8177-2571189ad9d6.jpg)
